test/python/result/test_marginal_counts.py

```diff
@@ -0,0 +1,51 @@
+# This code is part of Qiskit.
```

Comment:
There are already a lot of `marginal_counts` tests in `test/python/result/test_result.py` - can you put these tests with them in that format to keep everything together?